core/src/main/java/org/jboss/weld/vertx/AsyncReferenceImpl.java

```diff
@@ -46,19 +45,12 @@
  *
  * <ul>
  * <ol>
- * holds a special creational context so that we're able to correctly destroy dependent bean instances
```

Comment:
Technically, it still holds the CC, but now via `WeldInstance`, right?

Reply:
Exactly!

Reply:
So we might as well retain this part of doc and just alter it?

Reply:
Well, these are just implementation notes and since the behavior is now implicit I don't think we need to mention this.